<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="WL" ignored="0" name="Test" tests="10" failures="2" timestamp="12 三月 2018 06:39:24 GMT" time="96.669" errors="0">
  <testcase name="function1" time="25.595" classname="ftnt.qa.autotest.ui.fortisiem.pageobject.test">
    <skipped/>
  </testcase> <!-- function1 -->
  <testcase name="function1" time="14.668" classname="ftnt.qa.autotest.ui.fortisiem.pageobject.test">
    <skipped/>
  </testcase> <!-- function1 -->
  <testcase name="function1" time="17.952" classname="ftnt.qa.autotest.ui.fortisiem.pageobject.test">
    <skipped/>
  </testcase> <!-- function1 -->
  <testcase name="function1" time="36.569" classname="ftnt.qa.autotest.ui.fortisiem.pageobject.test">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at ftnt.qa.autotest.ui.fortisiem.pageobject.test.function1(test.java:23)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- function1 -->
  <testcase name="function2" time="0.001" classname="ftnt.qa.autotest.ui.fortisiem.pageobject.test"/>
  <testcase name="function3" time="0.0" classname="ftnt.qa.autotest.ui.fortisiem.pageobject.test">
    <skipped/>
  </testcase> <!-- function3 -->
  <testcase name="function3" time="0.001" classname="ftnt.qa.autotest.ui.fortisiem.pageobject.test">
    <skipped/>
  </testcase> <!-- function3 -->
  <testcase name="function3" time="0.0" classname="ftnt.qa.autotest.ui.fortisiem.pageobject.test">
    <skipped/>
  </testcase> <!-- function3 -->
  <testcase name="function3" time="0.001" classname="ftnt.qa.autotest.ui.fortisiem.pageobject.test">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at ftnt.qa.autotest.ui.fortisiem.pageobject.test.function3(test.java:33)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- function3 -->
  <testcase name="function4" time="0.001" classname="ftnt.qa.autotest.ui.fortisiem.pageobject.test"/>
</testsuite> <!-- Test -->
